pt_miniscreen/pages/root/projects/enums.py

```diff
@@ -0,0 +1,16 @@
+from enum import auto, Enum
+
+
+class ProjectState(Enum):
+    IDLE = auto()
+    STARTING = auto()
+    RUNNING_WITH_INSTRUCTIONS_IN_SCREEN = auto()
+    RUNNING_WITH_CLEAR_CREEN = auto()
```

Comment:
ok this makes more sense but i think typo 🙂 
```suggestion
    RUNNING_WITH_CLEAR_SCREEN = auto()
```